plugins/modules/keycloak_user.py

```diff
@@ -0,0 +1,498 @@
+        description:
+            - list user access.
+        required: false
+        type: dict
+    disableable_credential_types:
+        description:
+            - list user Credential Type.
+        default: []
+        type: list
+        elements: str
+        aliases:
+            - disableableCredentialTypes
+    origin:
+        description:
+            - user origin.
+        required: false
+        type: str
+    self:
+        description:
+            - user self administration.
+        required: false
+        type: str
+    state:
+        description:
+            - Control if the user must exists or not
+        choices: [ "present", "absent" ]
+        default: present
+        type: str
+    force:
+        description:
+            - If C(true), allows to remove user and recreate it.
+        type: bool
+        default: false
+extends_documentation_fragment:
+    - community.general.keycloak
+    - community.general.attributes
+attributes:
+    check_mode:
+        support: full
+    diff_mode:
+        support: full
+notes:
+    - The module does not modify the user ID of an existing user.
+author:
+    - Philippe Gauthier (@elfelip)
+'''
+
+EXAMPLES = '''
+- name: Create a user user1
+  community.general.keycloak_user:
+    auth_keycloak_url: http://localhost:8080/auth
+    auth_username: admin
+    auth_password: password
+    realm: master
+    username: user1
+    firstName: user1
+    lastName: user1
+    email: user1
+    enabled: true
+    emailVerified: false
+    credentials:
+        - type: password
+          value: password
+          temporary: false
+    attributes:
+        - name: attr1
+          values:
+            - value1
+          state: present
+        - name: attr2
+          values:
+            - value2
+          state: absent
+    groups:
+        - name: group1
+          state: present
+    state: present
+
+- name: Re-create a User
+  community.general.keycloak_user:
+    auth_keycloak_url: http://localhost:8080/auth
+    auth_username: admin
+    auth_password: password
+    realm: master
+    username: user1
+    firstName: user1
+    lastName: user1
+    email: user1
+    enabled: true
+    emailVerified: false
+    credentials:
+        - type: password
+          value: password
+          temporary: false
+    attributes:
+        - name: attr1
+          values:
+            - value1
+          state: present
+        - name: attr2
+          values:
+            - value2
+          state: absent
+    groups:
+        - name: group1
+          state: present
+    state: present
+    force: true
+
+- name: Remove User
+  community.general.keycloak_user:
+    auth_keycloak_url: http://localhost:8080/auth
+    auth_username: admin
+    auth_password: password
+    realm: master
+    username: user1
+    state: absent
+'''
+
+RETURN = '''
+user:
+  description: JSON representation for the user.
+  returned: on success
+  type: dict
+msg:
+  description: Message if it is the case
```

Comment:
```suggestion
  description: Message if it is the case.
```
If **what** is the case?